addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
> I agree it should be done correctly but you are not changing the backwards version. Period.

You do stop talking in this way with me! Who do you think you are that this is on you taking this decision? We can talk about this in the team but this is for sure not your decision. Did you get this?